homeassistant/components/simplefin/entity.py

```diff
@@ -0,0 +1,39 @@
+"""SimpleFin Base Entity."""
+
+from simplefin4py import Account
+
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import SimpleFinDataUpdateCoordinator
+
+
+class SimpleFinEntity(CoordinatorEntity[SimpleFinDataUpdateCoordinator]):
+    """Define a generic class for SimpleFIN entities."""
+
+    _attr_attribution = "Data provided by SimpleFIN API"
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: SimpleFinDataUpdateCoordinator,
+        description: EntityDescription,
+        account: Account,
+    ) -> None:
+        """Class initializer."""
+        super().__init__(coordinator=coordinator)
+        self.entity_description = description
+        self._account_id = account.id
+
+        self._attr_unique_id = f"account_{account.id}_{description.key}".lower()
```

Comment:
Why the lower

Reply:
I thought it looked better... 🤷️ ... I'll nix it.